--- zope_interface/setup.py     2017-06-21 15:49:50.470516337 -0400
+++ patched/setup.py    2017-06-21 15:51:07.699009097 -0400
@@ -46,7 +46,7 @@
         try:
             build_ext.build_extension(self, ext)
         except (CCompilerError, DistutilsExecError, OSError) as e:
-            self._unavailable(e)
+            raise e #: Force failure
 
     def _unavailable(self, e):
         print('*' * 80)
